src/server-entry.ts

```diff
@@ -7,10 +8,16 @@ import mwbot from 'mwbot';
 import EntityInitializer from './common/EntityInitializer';
 import getChildComponents from './common/getChildComponents';
 
-factory.setLanguageTranslationRepository( new WikibaseApiLanguageRepository() );
+const WIKIBASE_REPO_API = config.getWikibaseRepoApi(); // TODO test
```

Comment:
I don't particularly like this, and - as it stands - does not even help us to mock the value in tests (now dynamically setting the value from config there) but I see this as a separate challenge.